src/Sylius/Behat/Resources/config/services/contexts/ui.xml

```diff
@@ -322,10 +322,15 @@
             <argument type="service" id="sylius.behat.page.admin.impersonate_user" />
         </service>
 
+        <service id="sylius.behat.context.ui.admin.removing_products" class="Sylius\Behat\Context\Ui\Admin\RemovingProductContext">
```

Comment:
I would be for using here FQCN as a service id, as we do for new contexts, for example, for the catalog promotion